### ISSUE TITLE
Powershell Terminal part of the line disappear. 

### ISSUE THREAD

**Issue by aneo13** [Microsoft/vscode#34165]
- VSCode Version: 1.16
- OS Version: Win2K12 R2

I was trouble reproducing this issue on Win10 but what I can confirm this is issue with Active TAB in Powershell Terminal and it only appears in VSCode. When I run my window maximize the terminal output looks good but if I resize the window the the output is cut and no longer available for copy. And this apply's only to visible part of active TAB. 

1. example line
![image](https://user-images.githubusercontent.com/28782562/30293724-cbbacfa8-973a-11e7-8eb0-1184a9a159bd.png)
2. Resize
![image](https://user-images.githubusercontent.com/28782562/30293892-5c9014ca-973b-11e7-9547-871158dc6549.png)
3. maximize window again 
![image](https://user-images.githubusercontent.com/28782562/30293905-6bffcac2-973b-11e7-969a-cc6e17b6f3a5.png)

What is also strange this effect only apply to visible lines. Lines in buffer are intact as you can see the line UI. The other problem that might or might not be connected sometimes I'm missing scroll bar in Terminal and just need to use arrow (minimize maximize panel size) and scroll bar is back.

**Comment by aneo13** [comment]
My college replicated this issue on Windows 2k12 R2. Running VSCode 1.15.1 with Powershell Plugin only. Also I did try to reinstalling and deleting %appdata%\Code folder with no success of resolving the issue. I did also try to use older vscode 1.12 but issue still exist.

**Comment by aneo13** [comment]
What I'm guessing is happening is when I resize the window the buffersize is changing and is cutting the output. What is double strange that although  I set high buffer to 7000 I’m not able to display in vscode 3339 lines. 
1.	Looks like there is no possibility to set fix buffer width size and vscode adjust the width buffer automatically according  to current window size
2.	Looks like also vscode is not using buffer at all. As if I set high buffer to 200 I still can see way more lines than 200. 

![image](https://user-images.githubusercontent.com/28782562/30419751-c3d83650-9937-11e7-9c60-16ec19d248b0.png)
